lib/kubernetes-deploy/statsd.rb

```diff
@@ -22,5 +22,30 @@ def self.build
       end
       ::StatsD.backend
     end
+
+    module MeasureMethods
+      def measure_method(method_name, metric = nil)
```

Comment:
This is based on a [Shopify core PR](https://github.com/Shopify/shopify/pull/152598/files) that I found in looking up how to use dynamic tags in statd-instrument (which isn't possible).

Reply:
The gem already supports [tags](https://github.com/Shopify/statsd-instrument#tags) as well as [dynamic metric names](https://github.com/Shopify/statsd-instrument#dynamic-metric-names). Maybe we should just add this feature to the gem (which is owned by Shopify)?